src/AppBundle/Entity/ConnectionRepository.php

```diff
@@ -153,4 +153,24 @@ public function isUserConnectionExists(User $user1, User $user2)
 
         return $qb->getQuery()->getSingleScalarResult()? true: false;
     }
-}
+
+    /**
+     * @param $days
+     * @param $numSent
+     * @return array
+     */
+    public function findCreatedDaysAgo($days, $numSent)
+    {
+        return $this
+            ->createQueryBuilder('c')
+            ->where('DATE_DIFF(:now, c.createdAt) >= :days')
```

Comment:
With the `>=`-operator we will send emails to __all__ connections ever made when this is run for the first time. Is this by design @mikaelohren?